test/lit/wasm-split/jspi.wast

```diff
@@ -18,15 +18,15 @@
 
  ;; PRIMARY:      (import "env" "__load_secondary_module" (func $import$__load_secondary_module (param externref)))
 
- ;; PRIMARY:      (import "placeholder" "0" (func $placeholder_0 (param i32) (result i32)))
+ ;; PRIMARY:      (import "placeholder" "0" (func $fimport$1 (param i32) (result i32)))
```

Comment:
This looks like a loss of a useful name. Perhaps it's better to keep it?

Reply:
It's useful in tests, but it's not useful for production. Are tests enough motivation?

Reply:
I think so. In production no names are shipped anyhow so adding more names in non-production builds has little downside, so long as they are meaningful, which looks like the case here.